commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ProductAttributeDocumentation.java

```diff
@@ -104,7 +113,21 @@
 
 
 
- <h3 id="attribute-table-creation">Creating a table of attributes</h3>
+ <h3 id="attribute-table-creation">Creating a table of attributes using {@link io.sphere.sdk.products.attributes.DefaultProductAttributeFormatter}</h3>
+
+ <p>The most convenient way of creating a table of attributes is using a subclass of {@link io.sphere.sdk.products.attributes.DefaultProductAttributeFormatter}, since you can rely on defaults and just override the behaviour you want to specify. To initialize the class you need the supported locales of the user viewing the table and the cached {@link io.sphere.sdk.producttypes.ProductType}s of the commercetools project. Remember that you can fetch them all using {@link io.sphere.sdk.queries.QueryExecutionUtils}.</p>
+
+ <p>One example for a subclass:</p>
+
+ {@include.example io.sphere.sdk.products.attributes.ProjectNameProductAttributeFormatter}
+
+ <p>The example in action:</p>
+
+ {@include.example io.sphere.sdk.products.attributes.DefaultProductAttributeFormatterDemo}
+
+
+
+ <h3 id="attribute-table-creation-pattern-matching">Creating a table of attributes (old way)</h3>
```

Comment:
Hmm, it is very hard to use, I would always go with the new way. I don't see a use case.